web/app/themes/dp-timber-theme/index.php

```diff
@@ -16,7 +16,8 @@
 $context          = Timber::context();
 $context['posts'] = Timber::get_posts();
 $context['foo'] = 'Shayaan';
-$templates        = array( 'index.twig' );
+$context['qux'] = 'lorem ipsum doller si amut  ';
```

Comment:
what's the purpose of this varibale?

Reply:
I will change the variable name to be more descriptive

Reply:
I've updated the code, please review it